src/internal/direntry.rs

```diff
@@ -142,9 +142,10 @@ impl DirEntry {
         // Spec say this is suppose to be zero for DirEntries
         // but some cfb implementations set start_sector to FREE_SECTOR instead
```

Comment:
Can we update this comment as well?  (e.g. "...some cfb implementations set start_sector to FREE_SECTOR or END_OF_CHAIN instead.")